musileko/client/gl/texture.zig

```diff
@@ -144,45 +142,69 @@ pub const PixelFormat = struct {
     pub fn sizedFormat(comptime self: Self) c_uint {
```

Comment:
The changes to this switch and the associated changes to PixelComponent and PixelChannels are to better match up with the full range of GL supported texture formats:

https://www.khronos.org/opengl/wiki/Image_Format#Color_formats